docs/other-api/dev-v2.md

```diff
@@ -256,15 +256,10 @@ To use [Mock Service Worker][msw] in development, you'll need to:
 1. Run MSW as part of your app server
 2. Configure MSW to not mock internal "dev ready" messages to the dev server
 
-For example, if you are using [binode][binode] to integrate with MSW,
-make sure that the call to `binode` is within the `remix dev -c` subcommand.
-That way, the MSW server will have access to the `REMIX_DEV_HTTP_ORIGIN` environment variable:
-
 ```json filename=package.json
 {
   "scripts": {
-    "dev": "remix dev -c 'npm run dev:app'",
-    "dev:app": "binode --require ./mocks -- @remix-run/serve:remix-serve ./build"
+    "dev": "NODE_OPTIONS='--require ./mocks' remix dev",
```

Comment:
I like the `NODE_OPTIONS` approach better for beginners, so overall like this change. But couple issues I see here:

1. Need to add `cross-env` for Windows support
2. This applies the `NODE_OPTIONS` to both `remix dev` (dev server) and the underlying app server (in this case, `remix-serve`, since the `-c` flag was omitted) which causes both processes to require the mocks even though only the app server needs it.

```suggestion
    "dev": "remix dev -c \"npm run dev:app\"",
    "dev:app": "cross-env NODE_OPTIONS=\"--require ./mocks\" remix-serve ./build"
```